app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -395,4 +398,13 @@ def add_secret_to_pod_def(pod_def, inspector_admin_secret_name)
   def inspector_image
     'docker.io/openshift/image-inspector:2.0'
   end
+
+  def inspector_proxy_env_variables
+    settings = ext_management_system.custom_attributes
+    PROXY_ENV_VARIABLES.each_with_object([]) do |st, env|
```

Comment:
`no_proxy` means a list of hostnames that don't need any proxy. Those three are not mutually exclusive. It would mean that we have hostnames that don't use any proxy and the other hostnames will use http or https proxy according to what is defined (Actually `no_proxy` has no meaning if it is *not* accompanied by at least one of the other two) 
I will change the query.